src/hoc/ModifiedBackend.ts

```diff
@@ -3,7 +3,7 @@ import { HTML5Backend } from 'react-dnd-html5-backend'
 import { TouchBackend } from 'react-dnd-touch-backend'
 
 const shouldIgnoreTarget = (target: any) => {
-  return target.closest('.ProseMirror') !== null
+  return target.closest('.tiptap') !== null
```

Comment:
Any particular reason for changing `ProseMirror` class to `tiptap`?

Reply:
they both lie in the same document. So, I think tiptap is more recognizable, so.